content/applications/finance/fiscal_localizations/colombia.rst

```diff
@@ -83,51 +100,70 @@ Next, configure the :guilabel:`Fiscal Information` in the :guilabel:`Sales & Pur
+following configuration:
 
-Enable the :guilabel:`Test mode` checkbox to connect with the Carvajal testing environment.
+- Select :guilabel:`DIAN: Free Service` as the :guilabel:`Electronic Invoicing Provider`.
+- Configure the :guilabel:`Operation Modes` for the respective types of documents (*electronic
+  invoices* or *support documents*) to be generated from Odoo:
 
-Once Odoo and Carvajal are fully configured and ready for production, deactivate the :guilabel:`Test
-mode` checkbox to use the production database.
+  - :guilabel:`Software Mode`: the type of document to be generated with the operation mode.
+  - :guilabel:`Software ID`: the ID generated by |DIAN| for the specific operation mode.
+  - :guilabel:`Software PIN`: the PIN selected in the operation mode configuration in the |DIAN|
+    portal.
+  - :guilabel:`Testing ID`: the testing ID generated by |DIAN| and obtained after testing the
+    operation mode.
 
-.. image:: colombia/carvajal-configuration.png
-   :align: center
-   :alt: Configure credentials for Carvajal web service in Odoo.
+- Configure the available :guilabel:`Certificates` to sign your electronic documents:
 
-.. important::
-   :guilabel:`Test mode` must **only** be used on duplicated databases, **not** the production
-   environment.
+  - :guilabel:`Name`: the name of the certificate.
+  - :guilabel:`Certificate`: upload the certificate file in PEM format. After uploading a valid
+    certificate file, the following fields display:
 
-Report data configuration
--------------------------
+    - :guilabel:`Private Key`: Select an existing or create a new a private key by typing in a name
+      for the key, and select :guilabel:`Create and edit...` to display the :guilabel:`Create
+      Private Key` wizard. Then, upload a valid :guilabel:`Key file` and :guilabel:`Save & Close`.
+
+.. image:: colombia/dian-credentials-configuration.png
+   :alt: Colombian electronic invoicing credentials configured.
 
-Report data can be defined for the fiscal section and bank information of the PDF as part of the
-configurable information sent in the XML.
+.. note::
+   In a multi-company database, it is possible to have a different certificate for each company.
+
+.. _colombia/dian-environment-config:
+
+|DIAN| environment configuration
+--------------------------------
+
+The |DIAN| electronic invoicing module offers three different |DIAN| environments to connect with,
+each can be enabled from :menuselection:`Accounting --> Configuration --> Settings` and scroll to
+the :guilabel:`Colombian Electronic Invoicing` section:
 
-Navigate to :menuselection:`Accounting --> Configuration --> Settings`, and scroll to the
-:guilabel:`Colombian Electronic Invoicing` section, in order to find the :guilabel:`Report
-Configuration` fields. Here the header information for each report type can be configured.
+- **Certification environment**: This environment is useful to pass the |DIAN| certification
```

Comment:
Hi @afma-odoo! I disagree that these environment titles should use a :guilabel: because these titles are not used anywhere in the UI, rather, they are titles used to describe a specific configuration of fields in the UI.

For example, ``` :guilabel:`Test environment` ``` should have a :guilabel: since it is UI text. Let me know if you agree? :)

<img width="985" alt="Screenshot 2024-11-08 at 3 32 32 PM" src="https://github.com/user-attachments/assets/0b80049f-14c2-4e66-afc3-e2663a5a6344">